### ISSUE TITLE
[BUG] For printing in portrait, parts of the legend are missing

### ISSUE THREAD

**Issue by UJehle** [goat-community/goat#2065]
### Summary 📃

The last category of the "Local accessibility" Heatmal legend, as well as the AOIs are missing in the legend.
![grafik](https://user-images.githubusercontent.com/49870195/226972256-011b292d-b4a4-46af-bf4a-8345ab27add4.png)

### Steps to reproduce 👣

1.
2.
3.
4.

**Comment by EPajares** [comment]
The legend is to big it overflows. I think there is no fix to this in the current version

**Comment by EPajares** [comment]
AOIs we should still add

**Comment by EPajares** [comment]
@Ebubeker could you check if the AOIs are in the legend and if not add them next to the POIs

**Comment by Ebubeker** [comment]
Yes they are but the overflow.
